src/sagemaker/jumpstart/hub/parser_utils.py

```diff
@@ -19,12 +19,11 @@
 
 
 def camel_to_snake(camel_case_string: str) -> str:
-    """Converts camelCaseString or UpperCamelCaseString to snake_case_string."""
-    snake_case_string = re.sub("(.)([A-Z][a-z]+)", r"\1_\2", camel_case_string)
-    if "-" in snake_case_string:
-        # remove any hyphen from the string for accurate conversion.
-        snake_case_string = snake_case_string.replace("-", "")
-    return re.sub("([a-z0-9])([A-Z])", r"\1_\2", snake_case_string).lower()
+    """Converts PascalCase to snake_case_string using a regex.
```

Comment:
Inconsistent regex and method name.

Careful with renaming the method though, it would be backward incompatible.

Reply:
True, let me revert this to also cover camelCase as well